pom.xml

```diff
@@ -110,10 +110,10 @@
 						</configuration>
 				        <executions>
 							<execution>
-							<id>validate-java-code-formatting</id>
-							<phase>validate</phase>
+							<id>format-java-code</id>
+							<phase>install</phase>
```

Comment:
better format the code before the compilation and tests execution for just in case

Reply:
Fixed. Will do this in the `process-sources` phase

Reply:
better use `validate`

Reply:
It's not good to use `validate` for such cases. If there is any source generation, `validate` would be too early to format